server/accounts/src/index.ts

```diff
@@ -100,27 +100,27 @@ async function createAccount (db: Db, email: string, password: string): Promise<
 async function updateAccount (db: Db, email: string, password: string, newPassword: string, secondFactorEnabled: boolean, clientSecret: string, secondFactorCode: string): Promise<AccountInfo> {
   let account = await getAccount(db, email)
 
-  if (!account) {
+  if (account == null) {
```

Comment:
`===` here and below

Reply:
Nope, getAccount() return `| null` so check is correct here. it uses mongo API and it return null.

Reply:
Yeah, but is it intentional to use `==` here?

Reply:
Fixed.